app/services/message_sender.rb

```diff
@@ -8,11 +8,16 @@ def initialize(employee, message)
 
   def run
     configure_slack
-    channel_id = SlackChannelIdFinder.new(employee.slack_username, client).run
 
-    if channel_id
+    if employee.slack_channel_id.nil?
```

Comment:
nice! this logic all looks good. can we write a test that verifies its behavior, though? something in `spec/services/message_sender_spec.rb` ? If you want pointers on that, I am happy to help!